weblate/checks/format.py

```diff
@@ -439,6 +462,14 @@ class LuaFormatCheck(BasePrintfCheck):
     description = _("Lua format string does not match source")
 
 
+class SchemeFormatCheck(BasePrintfCheck):
+    """Check for Scheme format string."""
+
+    check_id = "scheme_format"
+    name = _("Scheme format")
+    description = _("Scheme format string does not match source")
```

Comment:
You will probably have to override `normalize` and `format_string` methods here as well.